src/abstract/AJSingleVaultTerminal.sol

```diff
@@ -225,10 +227,11 @@ abstract contract AJSingleVaultTerminal is AJPayoutRedemptionTerminal {
 
     function _targetLocalBalanceDelta(Vault storage _vault, int256 _change)
         internal
+        view
         returns (int256 delta)
     {
         // Get the amount of assets in the vault
-        // TODO: Should we use `convertToAssets` or `previewRedeem` here?
+        // previewRedeem calls convertToAssets internally as well so directly using convertToAssets and save gas
```

Comment:
A major difference between convertToAssets and previewRedeem is that previewRedeem includes the fees and slippage, where convertToAssets is a more simplistic conversion